src/js/models/LottoTicket.js

```diff
@@ -0,0 +1,21 @@
+import { generateLottoNumbers } from '../generateLottos.js';
+import { isAlreadyExist } from '../validators.js';
+
+export default class LottoTicketModel {
+  constructor() {
+    this.winningNumbers = []; // number[];
+  }
+
+  generate(originLottos) {
+    let hasSameLotto = true;
+    let newLottoNumbers = generateLottoNumbers();
+    while (hasSameLotto) {
+      if (!isAlreadyExist(originLottos.concat([newLottoNumbers]))) {
+        hasSameLotto = false;
+      } else {
+        newLottoNumbers = generateLottoNumbers();
+      }
+    }
+    this.winningNumbers = newLottoNumbers;
```

Comment:
```suggestion
    const newLottoNumbers = generateLottoNumbers();
    if (isAlreadyExist([ ...originLottos, newLottoNumbers ])) {
      this.generate(originLottos);
    } else {
      this.winningNumbers = newLottoNumbers;
    }
```
이렇게 재귀적으로 표현하는 방법도 있답니다!

Reply:
[b9a7eab](https://github.com/next-step/js-lotto/pull/184/commits/b9a7eab476bf1b6c8ae4fac1e4e0cea1f14e1313)로 반영했습니다!
감사합니다! 재귀 활용해보려다가 조금 잘 안되어서 일단 위와 같이 구현했었는데ㅠ 넘 복잡하게 생각할 필요가 없었던 것 같습니다ㅠ